src/config/blocks/listing/listingVariations.js

```diff
@@ -118,6 +120,20 @@ const iosanitaListingVariations = [
     },
     cloneData: cloneBlock,
   },
+  {
+    id: 'simpleListTemplate',
+    isDefault: false,
+    title: 'Lista semplice',
+    template: SimpleListTemplate,
+    skeleton: SimpleListTemplateSkeleton,
+    schemaEnhancer: ({ schema, formData, intl }) => {
+      let pos = addDefaultOptions(schema, formData, intl);
+      addSimpleListTemplateOptions(schema, formData, intl, pos);
+      addLinkMoreOptions(schema, formData, intl);
+      return schema;
+    },
+    cloneData: cloneBlock,
+  },
```

Comment:
c'era gia il suo pezzo di configurazione che era stato commentato in attesa che fosse portato dentro questo template. 
a questo punto rimuovi il pezzo che c'è alla riga 279 che non serve piu